test/views/errors/UserInsolventErrorSpec.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.errors
+
+import messages.InsolventErrorMessages
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import views.ViewBaseSpec
+import views.html.errors.UserInsolventError
+
+class UserInsolventErrorSpec extends ViewBaseSpec {
+
+  val userInsolvent: UserInsolventError = injector.instanceOf[UserInsolventError]
+
+  "Rendering the unauthorised page" should {
```

Comment:
Can you change this to something like "Rendering the insolvent error page" as unauthorised page already exists and may cause confusion